src/main/webapp/app/entities/programming-exercise/services/programming-exercise.service.ts

```diff
@@ -37,6 +37,12 @@ export class ProgrammingExerciseService {
         return this.http.put(this.resourceUrl + '/' + exerciseId + '/squash-template-commits', { responseType: 'text' });
     }
 
+    importExercise(programmingExercise: ProgrammingExercise): Observable<EntityResponseType> {
```

Comment:
```suggestion
    importExercise(adaptedSourceProgrammingExercise: ProgrammingExercise): Observable<EntityResponseType> {
```

Reply:
add a comment what the hell you are doing here ;-)

Reply:
True :smile: I added some JSDoc here explaining the method and the parameter